pandamium_datapack/data/pandamium/functions/misc/spawnpoint/advancement.mcfunction

```diff
@@ -0,0 +1,5 @@
+advancement revoke @s only pandamium:spawnpoint_set_via_respawn_anchor
+advancement revoke @s only pandamium:spawnpoint_set_via_bed
+tag @s add spawnpoint_advancement

```

Comment:
You should use a scoreboard rather than a tag. Olex doesn't want tags used unless they're really necessary or removed within the same tick.
e.g. `scoreboard objectives add set_spawnpoint dummy` in startup, and `scoreboard players set @s set_spawnpoint 1` here etc.